### PR TITLE
Updating doc to suggest the use of Marionette Objects

### LINKED ISSUE #2638: Controllers usage message
In the docs the following is stated
>Warning: deprecated. The Controller object is deprecated. Instead of using the Controller
> class with the AppRouter, you should specify your callbacks on a plain Javascript object.

Should it be suggested to use a marionette object instead of/as well as a JavaScript object?

### PR DESCRIPTION
**Pull request by chrislaughlin**
Closes #2638

### CODE REVIEW

File: docs/marionette.controller.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Marionette.Controller
 
 > Warning: deprecated. The Controller object is deprecated. Instead of using the Controller
-> class with the AppRouter, you should specify your callbacks on a plain Javascript object.
+> class with the AppRouter, you should specify your callbacks on a plain Javascript object or a [Marionette Object](./marionette.object.md)
```
Comment (samccone) [inline docs/marionette.controller.md]: why the `./` ? I am worried this might break on the www

Comment (chrislaughlin) [inline docs/marionette.controller.md]: This is following the same pattern as line 49 and 55

Comment (samccone) [inline docs/marionette.controller.md]: :+1:

**Comment by samccone** [comment]
@chrislaughlin thanks so much for the PR :clap: keep them coming :heart:

**Pull request merged**